### PR TITLE
Add istioctl and helm3

### PR DESCRIPTION
**Pull request by agianey**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,9 +11,11 @@ RUN apt-get update \
     groff \
     jq \
     less \
+    openssl \
     python3 \
     python3-pip \
-    python3-dev
+    python3-dev \
+    vim
```
Comment (agianey) [inline Dockerfile:18]: Add vim so `kubectl edit` works

File: Dockerfile [hunk 2]
```diff
@@ -27,19 +29,27 @@ RUN go get -u -v sigs.k8s.io/aws-iam-authenticator/cmd/aws-iam-authenticator
 
 ARG DOCKER_VERSION=18.09.7
 RUN curl -L -o- https://download.docker.com/linux/static/stable/x86_64/docker-$DOCKER_VERSION.tgz | tar -zxvf - && cp docker/docker \
-    /usr/bin/docker && chmod 755 /usr/bin/docker
+    /usr/local/bin/docker && chmod 755 /usr/local/bin/docker
```
Comment (agianey) [inline Dockerfile:32]: Moved everything to `/usr/local/bin` for consistency

File: Dockerfile [hunk 3]
```diff
@@ -27,19 +29,27 @@ RUN go get -u -v sigs.k8s.io/aws-iam-authenticator/cmd/aws-iam-authenticator
...
+RUN curl -L -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/amd64/kubectl \
+    && chmod 755 /usr/local/bin/kubectl
 
 ARG HELM_VERSION=2.16.1
-RUN	curl -L -o- https://storage.googleapis.com/kubernetes-helm/helm-v$HELM_VERSION-linux-amd64.tar.gz | tar -zxvf - && cp linux-amd64/helm \
-	/bin/helm && chmod 755 /bin/helm && helm init --client-only
+RUN curl -L -o- https://storage.googleapis.com/kubernetes-helm/helm-v$HELM_VERSION-linux-amd64.tar.gz | tar -zxvf - && cp linux-amd64/helm \
+    /usr/local/bin/helm && chmod 755 /usr/local/bin/helm && helm init --client-only
+
+ARG HELM3_VERSION=3.3.0
```
Comment (agianey) [inline Dockerfile:42]: Add helm3

File: Dockerfile [hunk 4]
```diff
@@ -27,19 +29,27 @@ RUN go get -u -v sigs.k8s.io/aws-iam-authenticator/cmd/aws-iam-authenticator
...
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
+    && chmod 700 get_helm.sh && BINARY_NAME=helm3 ./get_helm.sh --version v$HELM3_VERSION
 
 ARG KOPS_VERSION=1.15.0
-RUN	curl -L -o /bin/kops https://github.com/kubernetes/kops/releases/download/$KOPS_VERSION/kops-linux-amd64 \
-	&& chmod 755 /bin/kops
+RUN curl -L -o /usr/local/bin/kops https://github.com/kubernetes/kops/releases/download/$KOPS_VERSION/kops-linux-amd64 \
+    && chmod 755 /usr/local/bin/kops
+
+ARG ISTIO_VERSION=1.6.2
```
Comment (agianey) [inline Dockerfile:50]: Add istioctl

File: Dockerfile [hunk 5]
```diff
@@ -27,19 +29,27 @@ RUN go get -u -v sigs.k8s.io/aws-iam-authenticator/cmd/aws-iam-authenticator
...
-RUN	curl -L -o /bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/amd64/kubectl \
-	&& chmod 755 /bin/kubectl
+RUN curl -L -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/amd64/kubectl \
+    && chmod 755 /usr/local/bin/kubectl
 
 ARG HELM_VERSION=2.16.1
-RUN	curl -L -o- https://storage.googleapis.com/kubernetes-helm/helm-v$HELM_VERSION-linux-amd64.tar.gz | tar -zxvf - && cp linux-amd64/helm \
-	/bin/helm && chmod 755 /bin/helm && helm init --client-only
+RUN curl -L -o- https://storage.googleapis.com/kubernetes-helm/helm-v$HELM_VERSION-linux-amd64.tar.gz | tar -zxvf - && cp linux-amd64/helm \
+    /usr/local/bin/helm && chmod 755 /usr/local/bin/helm && helm init --client-only
```
Comment (jcwilson) [inline Dockerfile:40]: maybe we can get some line breaks in here around the `&&`s for readability?

File: Dockerfile [hunk 6]
```diff
@@ -11,9 +11,11 @@ RUN apt-get update \
     groff \
     jq \
     less \
+    openssl \
```
Comment (jcwilson) [inline Dockerfile:14]: just curious, when engendered this?

Comment (agianey) [reply Dockerfile:14]: ahh it was either istioctl or the helm3 install

File: Dockerfile (hunk 1 above)
Comment (ptramsey) [reply Dockerfile:18]: Might make sense to specify vim-nox.  I think 'vim' is a virtual package; I'm not sure which gets pulled in by default when you install it and if it's one of the -gtk variants that may make this image a lot larger than it needs to be.

File: Dockerfile (hunk 3 above)
Comment (ptramsey) [reply Dockerfile:42]: Does anything use it yet?

Comment (agianey) [reply Dockerfile:45]: Nope, just allowing us to support the migration, and for people to use things that require it

**Pull request merged**